src/vtbackend/Screen.cpp

```diff
@@ -797,6 +837,35 @@ void Screen<Cell>::backspace()
         _cursor.position.column--;
 }
 
+template <typename Cell>
+CRISPY_REQUIRES(CellConcept<Cell>)
+void Screen<Cell>::setScrollSpeed(int speed)
+{
+    if (speed >= 9)
+    {
+        // Speed value 9 defined by spec to be at maximum speed.
+        _terminal.settings().smoothLineScrolling = {};
+        return;
+    }
+
+    std::array<float, 9> constexpr NumberOfLinesPerSecond = { {
+        1,  // 0
+        2,  // 1
+        9,  // 2 | defined by spec to be 18 lines per second
+        13, // 3
+        18, // 4 | defined by spec to be 18 lines per second
```

Comment:
This is not a big deal, but if you are revisiting this code, I'd suggest maybe using 3, 6, 9, 12, and 18 for these first 5 settings. That way you'd also be able to select the original speeds used on some of the early DEC terminals: 6 for the VT1xx/VT2xx models; 3, 6, and 12 for the VT340; and you still have 9 and 18 in their standard positions for the VT5xx models.

Reply:
I better take care of it now, before I forget: #1231 :)